### PR TITLE
cnitool: add support for CNI_ARGS

### PR DESCRIPTION
**Pull request by feiskyer**
This PR add `CNI_ARGS` support in cnitool, so as to facilitate plugins required this environment variable.

### CODE REVIEW

**Comment by feiskyer** [comment]
cc @squeed @dcbw

File: cnitool/cni.go [hunk 1]
```diff
@@ -27,13 +28,30 @@ const (
...
 	CmdDel = "del"
 )
 
+func parseArgs(args string) ([][2]string, error) {
+	var result [][2]string
+
+	pairs := strings.Split(args, ";")
+	for _, pair := range pairs {
+		kv := strings.Split(pair, "=")
+		if len(kv) != 2 {
```
Comment (dcbw) [inline cnitool/cni.go]: Should this be `if len(kv) != 2 || kv[0] == "" || kv[1] = "" {` ?

Comment (feiskyer) [inline cnitool/cni.go]: hmm, yep, will fix.

Comment (feiskyer) [inline cnitool/cni.go]: Fixed. PTAL

**Pull request merged**